### PR TITLE
Update and rename add-network.md to wallet-api.md

### PR DESCRIPTION
**Pull request by abdullahvenly**
Updated and added Wallet-API introduction.

### CODE REVIEW

**Comment by abdullahvenly** [comment]
Hi @kmurphypolygon 

Added the guide again as per your instructions.

Also added guide to side-menu and removed one of the guides from side-menu.

Kindly review, thnks.

**Comment by kmurphypolygon** [comment]
Hi there, please check if `custom-tokens.md` is removed intentionally from the menu, and properly from the docs if it is still there.

**Comment by abdullahvenly** [comment]
Hi @kmurphypolygon 

The `custom-tokens.md` was removed intentionally from the menu. I have also deleted it from the docs.

File: docs/tools/wallets/venly/wallet-api.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+!!! caution "Content disclaimer"
+    Please view the third-party content disclaimer [here](https://github.com/0xPolygon/polygon-docs/blob/main/CONTENT_DISCLAIMER.md).
+
+# Wallet API
+
+The Wallet API allows developers to interact with blockchain networks and offer wallet functionality to their users without having to build everything from scratch. This can include features like account creation, transaction management, balance inquiries, and more.
+
+- Welcome your users with custom wallet branding. You can customize the user interface to your taste.
+- You are completely in charge of the wallet user experience to optimize your user conversion. Get total freedom on UX and asset management with our Wallet API.
```
Comment (hsutaiyu) [inline docs/tools/wallets/venly/wallet-api.md]:
```suggestion
- You are completely in charge of the wallet user experience to optimize user conversion. Get total freedom with regard to UX and asset management with our Wallet API.
```

File: docs/tools/wallets/venly/wallet-api.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- You and your users have complete control over digital assets, without any third-party interference. Securely manage wallets with complete autonomy and privacy.
```
Comment (hsutaiyu) [inline docs/tools/wallets/venly/wallet-api.md]:
```suggestion
- You and your users have complete control over digital assets without any third-party interference. Securely manage wallets with complete autonomy and privacy.
```

File: docs/tools/wallets/venly/wallet-api.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+- In case of lost login details, you and your users can recover access to wallets with a security code or biometric verification.
```
Comment (hsutaiyu) [inline docs/tools/wallets/venly/wallet-api.md]:
```suggestion
- In the event of loss of login credentials, you and your users can recover access to wallets with a security code or biometric verification.
```

File: docs/tools/wallets/venly/wallet-api.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+![Wallet-API to create Polygon wallets](../../../img/tools/wallet/venly/07.png)
+
+## Key features
+
+| Features | Description |
+| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Wallet management           | Developers can use the API to create, manage, and secure wallets for their users.                                                                                           |
+| Transaction services        | The API can enable the initiation and monitoring of blockchain transactions.                                                                                                |
+| Token support               | It may allow the handling of various tokens and assets on supported blockchain networks.                                                                                    |
+| Blockchain interactions     | Developers can integrate functionalities like reading data from the blockchain or writing data to it, more commonly known as creating and interacting with Smart Contracts. |
```
Comment (hsutaiyu) [inline docs/tools/wallets/venly/wallet-api.md]:
```suggestion
| Blockchain interactions     | Developers can integrate functionalities like reading data from the blockchain or writing data to it, along with creating and interacting with smart contracts. |
```

**Review by hsutaiyu — APPROVED** [review]
Approving with suggestions

**Pull request merged**